src/popup/components/emoji-picker.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react'
+import { useState } from 'react'
+// import "emoji-mart/css/emoji-mart.css";
+import { Picker } from 'emoji-mart'
+
+//
+export const EmojiPicker = () => {
+  const [emojiPickerState, SetEmojiPicker] = useState(false)
+  const [message, SetMessage] = useState('')
```

Comment:
Couple thoughts here

Instead of maintaining the `message` state here, let's call the `onEmojiPicked` callback that the parent `ActionBar` should provide us so that it may use the emoji.


```ts
  const [pickerActive, setPickerActive] = useState(false)
  
  const emojiPicker = pickerActive && (
    <Picker 
      title="Pick your emoji…"
      emoji="point_up"
      onSelect={emoji => {
        onEmojiPicked(emoji.native)
        setPickerActive(false) // Close the picker once the emoji is selected
      }}
    />
  )
```